addons/mrp/i18n/mrp.pot

```diff
@@ -454,12 +454,6 @@ msgstr ""
 msgid "Add quality checks to your work orders"
 msgstr ""
 
-#. module: mrp
```

Comment:
you can remove .pot diff since it's for master

Reply:
Whoops + removed